TESTS/mbed_hal/common_tickers/main.cpp

```diff
@@ -213,14 +215,24 @@ void ticker_info_test(void)
     TEST_ASSERT(p_ticker_info->bits >= 8);
 }
 
+
+
 /* Test that ticker interrupt fires only when the ticker counter increments to the value set by ticker_set_interrupt. */
 void ticker_interrupt_test(void)
 {
-    uint32_t ticker_timeout[] = { 100, 200, 300, 500 };
+    uint32_t ticker_timeout[] = { 100, 200, 300, 500, 600, 700, 800, 900, 1000, 1100, 1200 };
+
+    const ticker_info_t *p_ticker_info = intf->get_info();
 
     overflow_protect();
 
     for (uint32_t i = 0; i < (sizeof(ticker_timeout) / sizeof(uint32_t)); i++) {
+
+        /* Skip timeout if less than max allowed execution time of set_interrupt() - 20 us */
+        if (TICKS_TO_US(ticker_timeout[i], p_ticker_info->frequency) < (MAX_FUNC_EXEC_TIME_US + DELTA_FUNC_EXEC_TIME_US)) {
+            continue;
```

Comment:
There is no extra print or something else ?
This test could go into this continue line without any real test...?

Reply:
Good point! Added check at the end of the test case to ensure that at least 3 timeouts were tested.